xapi.opam

```diff
@@ -43,6 +43,7 @@ depends: [
   "xenctrl"
   "xenstore"
   "yojson"
+  "alcotest"
```

Comment:
Hmm I think it's incorrect in xs-opam then, because `alcotest` is already there at the top, so now it's duplicated

Reply:
Yes - it's twice in xs-opam. So let's close this and correct it there.